lib/services/computeManagement2/package.json

```diff
@@ -12,7 +12,7 @@
     "Zavery, Amar <amzavery@microsoft.com>",
     "Sun, Carissa <caris@microsoft.com>"
   ],
-  "version": "1.0.0-preview",
+  "version": "2.0.0-preview",
```

Comment:
NPM uses semantic versioning in the form of `major.minor.patch`. When your package is in preview, I suppose it has a major version of `0` . Once your package graduates to use a major version, it is no longer in preview. I never understood the usage of `-preview` in versioning. If you're specs out of preview, then I thought the package should reflect that. No?